Box.V2/Managers/BoxFilesManager.cs

```diff
@@ -894,15 +894,16 @@ public async Task<BoxCollection<BoxComment>> GetCommentsAsync(string id, IEnumer
         /// <param name="minWidth">The minimum width of the thumbnail.</param>
         /// <param name="maxHeight">The maximum height of the thumbnail.</param>
         /// <param name="maxWidth">The maximum width of the thumbnail.</param>
+        /// <param name="extension">png or jpg with no "."</param>
         /// <param name="handleRetry">Specifies whether the method handles retries. If true, then the method would retry the call if the HTTP response is 'Accepted'. The delay for the retry is determined 
         /// by the RetryAfter header, or if that header is not set, by the constant DefaultRetryDelay.</param>
         /// <param name="throttle">Whether the requests will be throttled. Recommended to be left true to prevent spamming the server.</param>
         /// <returns>Contents of thumbnail as Stream.</returns>
-        public async Task<Stream> GetThumbnailAsync(string id, int? minHeight = null, int? minWidth = null, int? maxHeight = null, int? maxWidth = null, bool throttle = true, bool handleRetry = true)
+        public async Task<Stream> GetThumbnailAsync(string id, int? minHeight = null, int? minWidth = null, int? maxHeight = null, int? maxWidth = null, string extension="png", bool throttle = true, bool handleRetry = true)
```

Comment:
Adding the new parameter in the middle of the list would also be a breaking change.  If someone was passing parameters positionally (e.g. `GetThumbnailAsync("12345", "0", "0", "512", "512", false)`), this change would break them.  The new parameter should be moved to the end of the parameter list.